### PR TITLE
Bump version to 1.7.8

### PR DESCRIPTION
**Pull request by rail**

### CODE REVIEW

File: deploy/certified-metadata-bundle/cockroach-operator/latest/cockroach-operator.v1.7.8.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,381 @@
...
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                app: cockroach-operator
+            spec:
+              containers:
+              - args:
+                - feature-gates
+                - Upgrade=false,ResizePVC=true
```
Comment (keith-mcclellan) [inline deploy/certified-metadata-bundle/cockroach-operator/latest/cockroach-operator.v1.7.8.clusterserviceversion.yaml:278]: Are these right @chrislovecnm ?

File: deploy/certified-metadata-bundle/cockroach-operator/latest/cockroach-operator.v1.7.8.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,381 @@
...
+      - description: (Required) Database disk storage configuration
+        displayName: Data Store
+        path: dataStore
+      - description: 'SupportsAutoResize marks that a PVC will resize without restarting
+          the entire cluster Default: false'
+        displayName: PVC Supports Auto Resizing
+        path: dataStore.supportsAutoResize
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:booleanSwitch
+      - description: (Required) Container image information
```
Comment (keith-mcclellan) [inline deploy/certified-metadata-bundle/cockroach-operator/latest/cockroach-operator.v1.7.8.clusterserviceversion.yaml:65]: Why is this described as required when it's optional?  It's only required if cockroachDBVersion isn't set

File: deploy/certified-metadata-bundle/cockroach-operator/latest/cockroach-operator.v1.7.8.clusterserviceversion.yaml [hunk 3]
```diff
@@ -0,0 +1,381 @@
...
+    mediatype: image/png
+  install:
+    spec:
+      clusterPermissions:
+      - rules:
+        - apiGroups:
```
Comment (keith-mcclellan) [inline deploy/certified-metadata-bundle/cockroach-operator/latest/cockroach-operator.v1.7.8.clusterserviceversion.yaml:115]: does the operator-sa really still need to be a superuser? I thought we constrained this in an earlier PR?

File: deploy/certified-metadata-bundle/cockroach-operator/latest/cockroach-operator.v1.7.8.clusterserviceversion.yaml [hunk 4]
```diff
@@ -0,0 +1,381 @@
...
+                app: cockroach-operator
+            spec:
+              containers:
+              - args:
+                - feature-gates
+                - Upgrade=false,ResizePVC=true
+                - -zap-log-level
+                - info
+                env:
+                - name: RELATED_IMAGE_COCKROACH_v19_2_12
```
Comment (keith-mcclellan) [inline deploy/certified-metadata-bundle/cockroach-operator/latest/cockroach-operator.v1.7.8.clusterserviceversion.yaml:282]: we should no longer be including v19.2.12 as its officially EOL

File: deploy/certified-metadata-bundle/cockroach-operator/latest/cockroach-operator.v1.7.8.clusterserviceversion.yaml [hunk 5]
```diff
@@ -0,0 +1,381 @@
...
+              serviceAccountName: cockroach-operator-sa
+      permissions:
+      - rules:
+        - apiGroups:
+          - '*'
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        serviceAccountName: cockroach-operator-sa
```
Comment (keith-mcclellan) [inline deploy/certified-metadata-bundle/cockroach-operator/latest/cockroach-operator.v1.7.8.clusterserviceversion.yaml:354]: This is also related to the superuser comment above

**Review by keith-mcclellan — CHANGES_REQUESTED** [review]
there are still a number of issues on this CSV

File: deploy/certified-metadata-bundle/cockroach-operator/latest/cockroach-operator.v1.7.8.clusterserviceversion.yaml [hunk 6]
```diff
@@ -0,0 +1,381 @@
...
+        - apiGroups:
+          - policy
+          resources:
+          - poddisruptionbudgets/status
+          verbs:
+          - '*'
+        - apiGroups:
+          - security.openshift.io
+          resourceNames:
+          - nonroot
```
Comment (udnay) [inline deploy/certified-metadata-bundle/cockroach-operator/latest/cockroach-operator.v1.7.8.clusterserviceversion.yaml:246]: @chrislovecnm @alinadonisa does this need to be `anyuid` as well?

File: deploy/certified-metadata-bundle/cockroach-operator/latest/manifests/cockroach-operator.v1.7.8.clusterserviceversion.yaml [hunk 7]
```diff
@@ -0,0 +1,381 @@
...
+        - apiGroups:
+          - policy
+          resources:
+          - poddisruptionbudgets/status
+          verbs:
+          - '*'
+        - apiGroups:
+          - security.openshift.io
+          resourceNames:
+          - nonroot
```
Comment (udnay) [inline deploy/certified-metadata-bundle/cockroach-operator/latest/manifests/cockroach-operator.v1.7.8.clusterserviceversion.yaml:246]: Same question as above

File: deploy/certified-metadata-bundle/cockroach-operator/latest/cockroach-operator.v1.7.8.clusterserviceversion.yaml (hunk 6 above)
Comment (keith-mcclellan) [reply deploy/certified-metadata-bundle/cockroach-operator/latest/cockroach-operator.v1.7.8.clusterserviceversion.yaml:246]: afaik the operator container (which this SA governs) doesn't require root.  its just the init container to read the secrets

**Pull request closed without merging**